apps/meteor/app/error-handler/server/lib/RocketChat.ErrorHandler.ts

```diff
@@ -1,9 +1,14 @@
 import { Meteor } from 'meteor/meteor';
 import { Settings, Users, Rooms } from '@rocket.chat/models';
+import { throttledCounter } from '@rocket.chat/tools';
 
 import { settings } from '../../../settings/server';
 import { sendMessage } from '../../../lib/server';
 
+const incException = throttledCounter((counter) => {
```

Comment:
wdyt of ignoring this setting on the `watch.settings` watcher?

Reply:
the main issue is that we'd still receive it from mongo.. so we'd need to remove it on oplog/change stream query, which is not good 😬